src/CabanaPD_Solver.hpp

```diff
@@ -103,13 +104,16 @@ class SolverElastic
     using force_model_type = ForceModel;
     using force_type = Force<exec_space, force_model_type>;
     using comm_type = Comm<particle_type, typename force_model_type::base_model,
-                           typename force_model_type::thermal_type>;
+                           typename particle_type::thermal_type>;
```

Comment:
What happened here? "thermal_type" is now a particle type, not a force_type?

Reply:
They used to be the same type, but now they're not. Comm directly uses the particle data types, so it makes more sense to use that one anyway